docs/install_macosx.md

```diff
@@ -19,6 +19,17 @@ You can turn off SIP on El Capitan this way:
   * Close the Terminal app.
   * Click the  (Apple) menu and select Restart....
 
+Or you can turn off SIP on El Capitan this way(silicon):
```

Comment:
I would rephrase this to:
On Apple silicon, the process of disabling SIP is different: